pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -94,14 +94,15 @@ stdenv.mkDerivation ({
   patches = [
     ./env_var_for_system_dir.patch
   ] ++ lib.optionals (stdenv.isAarch64) [
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/archlinuxarm/PKGBUILDs/09c7fa0dc1d87922e3b464c0fa084df1227fca79/extra/firefox/arm.patch";
-      sha256 = "1vbpih23imhv5r3g21m3m541z08n9n9j1nvmqax76bmyhn7mxp32";
-    })
     (fetchpatch {
       url = "https://raw.githubusercontent.com/archlinuxarm/PKGBUILDs/09c7fa0dc1d87922e3b464c0fa084df1227fca79/extra/firefox/build-arm-libopus.patch";
       sha256 = "1zg56v3lc346fkzcjjx21vjip2s9hb2xw4pvza1dsfdnhsnzppfp";
     })
+    # https://phabricator.services.mozilla.com/D60667
+    (fetchpatch {
+      url = "https://hg.mozilla.org/mozilla-central/raw-rev/b3d8b08265b800165d684281d19ac845a8ff9a66";
+      sha256 = "sha256-94jlP07t9kC5t9f4y0pWQt4V/fa0SkZfTsFQfXg5miw=";
```

Comment:
This should be either `hash = "<sri-hash>"` or `sha256 = "<sha256-hash>`". Saying sha256 twice is redundant (and probably shouldn't be allowed...).